docs/enterprise/troubleshooting.md

```diff
@@ -0,0 +1,122 @@
+
+{% include toc.html %}
+
+A lakeFS Enterprise deployment has multiple moving parts that must all be deployed and configured correctly. This is especially true during initial setup. To help troubleshoot issues, both lakeFS and fluffy include the `flare` command.
+
+## The `flare` command
+
+#### Synopsis
+
+Both `lakefs` and `fluffy` include the flare command
+
+```
+lakefs flare [flags]
+fluffy flare [flags]
+```
+
+#### Flags
+{:.no_toc}
+
+```
+        --env-var-filename      the name of the file environment variables will be written to (default: lakefs-env.txt)
+        --include-env-vars      should environment variables be collected by flare (default: true)
+    -o, --output                Output path relative to the current path
+    -p, --package               Package generated artifacts into a .zip file (default: false)
+        --stdout                Output to stdout instead of files (default: false)
+        --zip-filename          The name of the zip file created when the -p option is used (default: lakefs-flare.zip)
+```
+
+#### Example Usage
+{:.no_toc}
+
+```shell
+# This will run flare with output set to stdout, which is redirected into a file
+$ ./lakefs flare --stdout > lakefs.flare
+# The same works for fluffy
+$ ./fluffy flare --stdout > fluffy.flare
+```
+
+## What Information Does the `flare` Command Collect?
+
+### Configuration
+
+Both lakeFS and fluffy allow configuration to be supplied in multiple ways: configuration file, environment variables, `.env` files, and command flags. The `flare` command collects the fully resolved final configuration used by the lakeFS/fluffy process.
+
+### Environment Variables
+
+When troubleshooting, it's important to get a view of the environment in which lakeFS/fluffy are running. This is especially true for container-based deployment environments, like K8s, where env vars are used extensively. The `flare` command collects environment variables with the following prefixes:
+
+- `LAKEFS_`
+- `FLUFFY_`
+- `HTTP_`
+- `HOSTNAME`
+
+### Sanitization and Secret Redaction
+
+Both configuration and env var include sensitive secrets. The `flare` command has a multi-step process for redacting secrets from what it collects. The `flare` command is able to detect the following types of secrets:
+
+- AWS static credentials
+- Azure storage keys
+- Basic HTTP auth username/password
+- JWTs
+- Bearer auth headers
+- GitHub tokens
+- Certificate private keys
+
+Aside from the specific secret type listed above, `flare` also has the ability to detect and redact generic high-entropy strings, which are likely to be secrets.
+
+Redacted secrets are replaced by a `SHA512` hash of the value. This allows comparing values (e.g., between lakeFS and fluffy) without exposing the actual values.
+
+Although we've taken great care to clean the information `flare` collects of secrets, the process isn't perfect. We recommend manually reviewing the `flare`'s output before sharing them.
+
+## Example Script
```

Comment:
I'm missing the part where we can send a url to users and tell them follow steps 1.2.3 and done. 
This part is probably it but lacking structure. 
So for start instead of **Example Script**  can we change the title to something more like "Usage - collect and send a flare"